modules/microshift-installing-from-rpm.adoc

```diff
@@ -1,21 +1,21 @@
 // Module included in the following assemblies:
 //
-// microshift/microshift-install-rpm.adoc 
+// microshift/microshift-install-rpm.adoc
 
 :_content-type: PROCEDURE
 [id="installing-microshift-from-rpm-package_{context}"]
-= Installing MicroShift from an RPM package 
+= Installing {product-title} from an RPM package
 
-Use the following procedure to install MicroShift from an RPM package. 
+Use the following procedure to install {product-title} from an RPM package.
 
-.Prerequisites 
+.Prerequisites
 
-* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met.
```

Comment:
No links in modules

Reply:
@mburke5678 thanks for catching these. I completely overlooked them.